contracts/Remittance.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity >=0.4.21 <0.6.0;
+
+import "./Pausable.sol";
+
+contract Remittance is Pausable {
+  address public owner;
```

Comment:
Which one is the real owner. This one, or the one in `Ownable`?

Reply:
My bad, I don't need `address public owner;` here, if I put it here, there are two instances of `owner` state on-chain I reckon.